src/main/java/src/ProductList.java

```diff
@@ -2,20 +2,91 @@
 
 import java.util.ArrayList;
 
+import static src.Product.*;
+
 public class ProductList {
-    private final String name;
-    private ArrayList<Product> listOfProducts;
+  private final String name;
+  private final ArrayList<Product> listOfProducts;
+
+  public ProductList(String name, ArrayList<Product> listOfProducts) {
+    this.name = sanitizeString(name);
+    this.listOfProducts = listOfProducts;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  private static String sanitizeString(String name) {
```

Comment:
Maybe change method name since you only use it to sanitize names.

Reply:
Fixed.